drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/webapp/test.html

```diff
@@ -0,0 +1,82 @@
+<!--
+  ~ Copyright 2019 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>SCESIM Editor Test</title>
+
+    <script>
+        function newDiagram() {
+            window.frames.editorFrame.contentWindow.gwtEditorBeans.get("ScenarioSimulationEditor").get().setContent("","");
+        }
+
+        function loadDiagram(diagram) {
+            console.log('loadDiagram ' + diagram);
+            window.frames.editorFrame.contentWindow.gwtEditorBeans.get("ScenarioSimulationEditor").get().setContent('', diagram);
+
+            //window.frames.editorFrame.contentWindow.loadFile();
+            //document.getElementById("download_btn").disabled = false;
+        }
+
+        function importDMN() {
+            window.frames.editorFrame.contentWindow.importDMN();
+            document.getElementById("download_btn").disabled = false;
+        }
+
+        function download() {
+                window.frames.editorFrame.contentWindow.gwtEditorBeans.get("ScenarioSimulationEditor").get().getContent().then(
+                    function(p) {
+                        var d = document.createElement('a');
+                        d.setAttribute('href', 'data:text/xml;charset=utf-8,' + encodeURIComponent(p));
+                        d.setAttribute('download', "diagram.sce");
```

Comment:
```suggestion
                        d.setAttribute('download', "diagram.scesim");
```
Please fix the extension name.